torch/_dynamo/backends/onnxrt.py

```diff
@@ -1,131 +1,6 @@
-import importlib
-import logging
-import os
-import tempfile
-
-import torch
-from .common import device_from_inputs, fake_tensor_unsupported
+from torch.onnx._internal.onnxruntime import has_onnxruntime, make_aot_ort
```

Comment:
should this file have any docstring to list maintainers, etc?

Reply:
Sure. Will do
```
# This backend is maintained by ONNX team. To direct issues
# to the right people, please tag related GitHub issues with `onnx`.
#
# Maintainers' Github IDs: wschin, thiagocrepaldi, BowenBao, abock
from torch.onnx._internal.onnxruntime import has_onnxruntime, make_aot_ort
from .registry import register_backend

```